src/openms/source/FILTERING/DATAREDUCTION/FeatureFindingMetabo.cpp

```diff
@@ -316,6 +335,12 @@ namespace OpenMS
     defaults_.setValue("remove_single_traces", "false", "Remove unassembled traces (single traces).");
     defaults_.setValidStrings("remove_single_traces", ListUtils::create<String>("false,true"));
 
+    //TODO does this make other parameters obsolete, or do we still want to support all different options?
+    defaults_.setValue("mz_scoring_by_elements", "false", "Use the m/z range of the assumed elements to detect isotope peaks. A expected m/z range is computed from the isotopes of the assumed elements. If enabled, this ignores 'mz_scoring_13C'");
```

Comment:
better to have the user select from a list of methods instead of exclusive flags